public_suffix_list.dat

```diff
@@ -13618,19 +13618,20 @@ syncloud.it
 
 // Synology, Inc. : https://www.synology.com/
 // Submitted by Rony Weng <ronyweng@synology.com>
-diskstation.me
 dscloud.biz
-dscloud.me
-dscloud.mobi
+direct.quickconnect.cn
```

Comment:
Could you explain more about why this name is needed, and the existing domains are insufficient? Effectively, every domain added becomes a cumulative cost to the Internet at large (that ships this list in a wide variety of applications), and so it's useful to understand.

Don't get me wrong, I [don't think it's great](https://github.com/sleevi/psl-problems) that this is the case, but that's the situation today, so it's useful to better understand individual rationale.

Reply:
Sure.

As `direct.quickconnect.to` is provided as one of our dns service and published in china, we are requested to provide a `.cn` TLD for policy from china administration.

Reply:
Thanks! For record keeping, do you have a reference to that policy you can point to?

Reply:
Sure.

If user would like to apply for an ICP filing, a valid domain name is needed ([ref](https://support.huaweicloud.com/intl/en-us/prepare-icp/icp_02_0002.html)). While our original domain direct.quickconnect.to is not a valid domain listed [here](http://xn--eqrt2g.xn--vuq861b/?spm=a2c63.p38356.0.0.3559939axsXaJV#) by MIIT(ministry of industry and information technology), a valid domain is required.

Reply:
Hi @sleevi 

Should I provide any further information for this pull request ?

Reply:
Thanks! That'll do.